integrationtests/tests/tests.rs

```diff
@@ -970,3 +971,53 @@ async fn rejoin_consensus_threshold_peers() -> Result<()> {
     })
     .await
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn ecash_can_be_recovered() -> Result<()> {
+    test(4, |fed, user_send, bitcoin, _, _| async move {
```

Comment:
You might be able to use 2 peers which makes the tests faster.

Reply:
Can you explain?

Reply:
I mean just changing the 4 to a 2 will reduce the size of the federation.  Usually I used 2 peers unless necessary.

Reply:
Oh. Thanks! #1131